documentation/API_documentation/Connectivity_API.md

```diff
@@ -0,0 +1,86 @@
+# Overview
+This API makes possible to query the connectivity status of user equipment.
+
+## 1\. Introduction
+<img src="resources/DeviceStatus_Connectivity_diagram.png">
+
+## 2\. Quick Start
+DeviceStatus API v0.2.0 exposes only one endpoint, which can be used to check connectivity status for user equipment. To do so user has to pass following parameters in request body:
+1. **ueId** - an object with four fields, each of them make possible to pass ueId in different format: externalId [1], msisdn, ipv4Addr and ipv6Add
+2. **eventType** - string value which determines the connectivity event to check its status, for now only "UE_ROAMING_STATUS" is supported.
+
```

Comment:
Just as a comment, uePort isn't a required input parameter and user doesn't "have to" send it but maybe we could mention it here as an optional.